src/adapters/MeetingsJSONAdapter.js

```diff
@@ -240,6 +248,7 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     );
     const rosterEvents$ = fromEvent(document, ROSTER_CONTROL);
     const settingsEvents$ = fromEvent(document, SETTINGS_CONTROL);
+    const cameraSwitcherEvents$ = fromEvent(document, SWITCH_CAMERA_CONTROL);
```

Comment:
better switchCameraEvents, so we use the verb (like shareEvents, joinEvents, leaveEvents)